doc/source/api/alleleAnnotations.rst

```diff
@@ -0,0 +1,93 @@
+
+Variant Annotation API
+!!!!!!!!!!!!!!!!!!!!!!
+
+See `Allele Annotation schema <../schemas/alleleAnnotations.html>`_ for a detailed reference.
+
+Introduction
+@@@@@@@@@@@@
+
+Variant alleles can be annotated by comparing them to other reference data sets
+using a variety of algorithms. A standard form of annotation is to compare 
+alleles to a transcript set and calculate the expected functional consequence 
+of the change ( e.g. a variant within a protein coding transcript may change the
+amino acid sequence of the resulting protein).
+
+This API supports the mining of variant annotations by region or genomic 
+feature and the filtering of the results by predicted functional effect.
+
+Variants Schema Entities
+@@@@@@@@@@@@@@@@@@@@@@@@
+
+The ``VariantAnnotation`` data model, is based on the results provided by variant 
+annotation programs such as VEP, SnpEff and Annovar and others, as well as the 
+VCF's `ANN format <http://snpeff.sourceforge.net/VCFannotationformat_v1.0.pdf>`_ . 
+
+
++---------------------+---------------------------------------------------------------------------------------------------------------------+
+| Record              | Description                                                                                                         |
++=====================+=====================================================================================================================+
+| VariantAnnotationSet| A VariantAnnotationSet record groups VariantAnnotation records. It represents the comparison of a VariantSet to a   |
+|                     | specified reference data set using specified algorithms. It holds information describing the software and reference |
```

Comment:
Both 'reference' and 'data set' have specific meanings within the schema - are we talking about precisely these objects?